Lodgify.Payments.Stripe.Application/Services/IStripeClient.cs

```diff
@@ -2,6 +2,6 @@
 
 public interface IStripeClient
 {
-    Task<Domain.Accounts.Account> CreateAccount(string country, string email, CancellationToken cancellationToken = default);
+    Task<Domain.Accounts.Account> CreateAccount(int userId, string country, string email, CancellationToken cancellationToken = default);
```

Comment:
How about using naming convention for asynchronous methods with Async postfix in the name? It is usually used in Lodgify everywhere except Controller methods.

Also, I propose to never use the default for cancellationToken to force developers to always explicitly provide it.

The proposal is for this method and all others across the whole solution.

```suggestion
    Task<Domain.Accounts.Account> CreateAccountAsync(int userId, string country, string email, CancellationToken cancellationToken);
```